GVFS/GVFS.Common/InstallerPreRunChecker.cs

```diff
@@ -128,6 +138,12 @@ protected virtual bool IsUnattended()
 
         protected virtual bool IsBlockingProcessRunning(out HashSet<string> processes)
         {
+            if (this.dryRun)

```

Comment:
Why is this here? I think dry-run should fail if a blocking process exists. The way I think about this is dry-run should be semantically identical to the real thing except that we actually run the installers.  We _should_ have dry run fail if there's a process hanging around, since this will be how we do functional tests, and our functional tests should give us an indication of expected reliability.

Reply:
Enabled pre-checks for dry-run.